### PR TITLE
[Program:GCI] Configure Pylint in travis.yml to check only for errors

### PR DESCRIPTION
**Pull request by daksha-aeer**
### Description
Travis file configured to check for only pylint errors and ignore warnings,refactor,convention and fatal class messages. For this **'--errors-only'** flag was added to pylint execution command.

### Type of Change:

- Code
- Quality Assurance

**Code/Quality Assurance Only**
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
1. Pylint help to know how to check only for errors
```sh
pylint --help
```
task65_2

2. Run pylint command with --errors-only
```sh
pylint ./app --errors-only
```
task65_1

**Output**
```sh
(venv) home@HP:~/Desktop/Dak/mentorship-backend$ pylint ./app --errors-only
************* Module app.api.dao.mentorship_relation
app/api/dao/mentorship_relation.py:177:19: E0602: Undefined variable 'CANT_ACCEPT_UNINVOLVED_MENTOR_RELATION' (undefined-variable)
************* Module app.database.db_types.JsonCustomType
app/database/db_types/JsonCustomType.py:5:21: E1101: Instance of 'SQLAlchemy' has no 'TypeDecorator' member (no-member)
app/database/db_types/JsonCustomType.py:8:11: E1101: Instance of 'SQLAlchemy' has no 'Text' member (no-member)
************* Module app.database.models.tasks_list
app/database/models/tasks_list.py:19:9: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/tasks_list.py:19:19: E1101: Instance of 'SQLAlchemy' has no 'Integer' member (no-member)
app/database/models/tasks_list.py:20:12: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/tasks_list.py:21:19: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/tasks_list.py:21:29: E1101: Instance of 'SQLAlchemy' has no 'Integer' member (no-member)
app/database/models/tasks_list.py:168:8: E1101: Instance of 'scoped_session' has no 'add' member (no-member)
app/database/models/tasks_list.py:169:8: E1101: Instance of 'scoped_session' has no 'commit' member (no-member)
app/database/models/tasks_list.py:173:8: E1101: Instance of 'scoped_session' has no 'delete' member (no-member)
app/database/models/tasks_list.py:174:8: E1101: Instance of 'scoped_session' has no 'commit' member (no-member)
************* Module app.database.models.user
app/database/models/user.py:20:9: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:20:19: E1101: Instance of 'SQLAlchemy' has no 'Integer' member (no-member)
app/database/models/user.py:23:11: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:23:21: E1101: Instance of 'SQLAlchemy' has no 'String' member (no-member)
app/database/models/user.py:24:15: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:24:25: E1101: Instance of 'SQLAlchemy' has no 'String' member (no-member)
app/database/models/user.py:25:12: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:25:22: E1101: Instance of 'SQLAlchemy' has no 'String' member (no-member)
app/database/models/user.py:28:20: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:28:30: E1101: Instance of 'SQLAlchemy' has no 'String' member (no-member)
app/database/models/user.py:31:24: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:31:34: E1101: Instance of 'SQLAlchemy' has no 'Float' member (no-member)
app/database/models/user.py:32:35: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:32:45: E1101: Instance of 'SQLAlchemy' has no 'Boolean' member (no-member)
app/database/models/user.py:35:15: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:35:25: E1101: Instance of 'SQLAlchemy' has no 'Boolean' member (no-member)
app/database/models/user.py:38:24: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:38:34: E1101: Instance of 'SQLAlchemy' has no 'Boolean' member (no-member)
app/database/models/user.py:39:30: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:39:40: E1101: Instance of 'SQLAlchemy' has no 'DateTime' member (no-member)
app/database/models/user.py:42:30: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:42:40: E1101: Instance of 'SQLAlchemy' has no 'Integer' member (no-member)
app/database/models/user.py:43:24: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:43:34: E1101: Instance of 'SQLAlchemy' has no 'Integer' member (no-member)
app/database/models/user.py:45:10: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:45:20: E1101: Instance of 'SQLAlchemy' has no 'String' member (no-member)
app/database/models/user.py:46:15: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:46:25: E1101: Instance of 'SQLAlchemy' has no 'String' member (no-member)
app/database/models/user.py:47:17: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:47:27: E1101: Instance of 'SQLAlchemy' has no 'String' member (no-member)
app/database/models/user.py:48:19: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:48:29: E1101: Instance of 'SQLAlchemy' has no 'String' member (no-member)
app/database/models/user.py:49:21: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:49:31: E1101: Instance of 'SQLAlchemy' has no 'String' member (no-member)
app/database/models/user.py:50:25: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:50:35: E1101: Instance of 'SQLAlchemy' has no 'String' member (no-member)
app/database/models/user.py:51:13: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:51:23: E1101: Instance of 'SQLAlchemy' has no 'String' member (no-member)
app/database/models/user.py:52:16: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:52:26: E1101: Instance of 'SQLAlchemy' has no 'String' member (no-member)
app/database/models/user.py:53:17: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:53:27: E1101: Instance of 'SQLAlchemy' has no 'String' member (no-member)
app/database/models/user.py:54:16: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:54:26: E1101: Instance of 'SQLAlchemy' has no 'String' member (no-member)
app/database/models/user.py:56:21: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:56:31: E1101: Instance of 'SQLAlchemy' has no 'Boolean' member (no-member)
app/database/models/user.py:57:26: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/user.py:57:36: E1101: Instance of 'SQLAlchemy' has no 'Boolean' member (no-member)
app/database/models/user.py:150:8: E1101: Instance of 'scoped_session' has no 'add' member (no-member)
app/database/models/user.py:151:8: E1101: Instance of 'scoped_session' has no 'commit' member (no-member)
app/database/models/user.py:155:8: E1101: Instance of 'scoped_session' has no 'delete' member (no-member)
app/database/models/user.py:156:8: E1101: Instance of 'scoped_session' has no 'commit' member (no-member)
************* Module app.database.models.mentorship_relation
app/database/models/mentorship_relation.py:31:9: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/mentorship_relation.py:31:19: E1101: Instance of 'SQLAlchemy' has no 'Integer' member (no-member)
app/database/models/mentorship_relation.py:34:16: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/mentorship_relation.py:34:26: E1101: Instance of 'SQLAlchemy' has no 'Integer' member (no-member)
app/database/models/mentorship_relation.py:34:38: E1101: Instance of 'SQLAlchemy' has no 'ForeignKey' member (no-member)
app/database/models/mentorship_relation.py:35:16: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/mentorship_relation.py:35:26: E1101: Instance of 'SQLAlchemy' has no 'Integer' member (no-member)
app/database/models/mentorship_relation.py:35:38: E1101: Instance of 'SQLAlchemy' has no 'ForeignKey' member (no-member)
app/database/models/mentorship_relation.py:36:21: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/mentorship_relation.py:36:31: E1101: Instance of 'SQLAlchemy' has no 'Integer' member (no-member)
app/database/models/mentorship_relation.py:37:13: E1101: Instance of 'SQLAlchemy' has no 'relationship' member (no-member)
app/database/models/mentorship_relation.py:40:13: E1101: Instance of 'SQLAlchemy' has no 'relationship' member (no-member)
app/database/models/mentorship_relation.py:44:20: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/mentorship_relation.py:44:30: E1101: Instance of 'SQLAlchemy' has no 'Float' member (no-member)
app/database/models/mentorship_relation.py:45:18: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/mentorship_relation.py:45:28: E1101: Instance of 'SQLAlchemy' has no 'Float' member (no-member)
app/database/models/mentorship_relation.py:46:17: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/mentorship_relation.py:46:27: E1101: Instance of 'SQLAlchemy' has no 'Float' member (no-member)
app/database/models/mentorship_relation.py:47:15: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/mentorship_relation.py:47:25: E1101: Instance of 'SQLAlchemy' has no 'Float' member (no-member)
app/database/models/mentorship_relation.py:49:12: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/mentorship_relation.py:49:22: E1101: Instance of 'SQLAlchemy' has no 'Enum' member (no-member)
app/database/models/mentorship_relation.py:50:12: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/mentorship_relation.py:50:22: E1101: Instance of 'SQLAlchemy' has no 'String' member (no-member)
app/database/models/mentorship_relation.py:52:20: E1101: Instance of 'SQLAlchemy' has no 'Column' member (no-member)
app/database/models/mentorship_relation.py:52:30: E1101: Instance of 'SQLAlchemy' has no 'Integer' member (no-member)
app/database/models/mentorship_relation.py:52:42: E1101: Instance of 'SQLAlchemy' has no 'ForeignKey' member (no-member)
app/database/models/mentorship_relation.py:53:17: E1101: Instance of 'SQLAlchemy' has no 'relationship' member (no-member)
app/database/models/mentorship_relation.py:100:8: E1101: Instance of 'scoped_session' has no 'add' member (no-member)
app/database/models/mentorship_relation.py:101:8: E1101: Instance of 'scoped_session' has no 'commit' member (no-member)
app/database/models/mentorship_relation.py:106:8: E1101: Instance of 'scoped_session' has no 'delete' member (no-member)
app/database/models/mentorship_relation.py:107:8: E1101: Instance of 'scoped_session' has no 'commit' member (no-member)
(venv) home@HP:~/Desktop/Dak/mentorship-backend$ 
```

3. Add this to .travis.yml

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged
- [ ] Update Postman API at /docs folder
- [ ] Update Swagger documentation and the exported file at /docs folder
- [x] Update requirements.txt

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,7 +7,9 @@ before_script:
  - chmod +x deploy.sh
 install:
   - pip3 install -r requirements.txt
+  - pip3 install pylint
```
Comment (isabelcosta) [inline .travis.yml]: by having pylint in the `requirements.txt` do you need this line, installing pylint?
I suggest removing this. what do you think? @daksha-aeer

**Review by isabelcosta — CHANGES_REQUESTED** [review]
Thank you so much for your contribution @daksha-aeer !
I left a comment for you to address

File: .travis.yml (hunk 1 above)
Comment (daksha-aeer) [reply .travis.yml]: Done!

**Review by Tlazypanda — APPROVED** [review]
Approved :)

**Comment by isabelcosta** [comment]
Thank you for your contribution @daksha-aeer !
We decided to close PRs related to this task so that we can have an issue, where a contributor can add this configuration and fix the problems that arise, from this new configuration.

**Pull request closed without merging**